Aakanksha/security w (#215)
NuevoFoundation/workshops: content/english/security-phishing/2-Phishing Basics/_index.md

---
title: "Phishing Basics"
description: ""
draft: false
weight: 2

> translationKey: "security-2"

Reviewer: ```suggestion
```

---

Our Secret Agent Alice explains to Bob about Phishing!!!

## What is Phishing?

> Just like while fishing, one uses a bait to catch fishes on a rod, malicious <b>hackers</b>

Reviewer: ```suggestion
Just like while fishing, one uses a bait to catch fishes on a rod, malicious **hackers**
```

> Just like while fishing, one uses a bait to catch fishes on a rod, malicious <b>hackers</b>
> use different types of bait to steal personal information. 

Reviewer: ```suggestion
Just like when fishing, one uses a bait to catch fish on a rod, malicious <b>hackers</b>
use different types of bait to steal personal information from unsuspecting people.
```

Phishing is a type of social engineering attack which is used to steal sensitive data, 
including login credentials, banking and credit card details. <br>

> It occurs when a <b>Hacker</b>,

Reviewer: ```suggestion
It occurs when a **Hacker**,
```

posing as a trusted entity, lures an individual using email, text message, or phone.<br> The recipient
is then tricked into clicking a malicious link, which can lead to the installation of a malicious
software, or revealing of sensitive data.

## What went wrong?

> Bob was enticed by an email from a hacker offering him a free trip to Bahamas to steal his personal

Reviewer: ```suggestion
Bob was enticed by an email from a hacker named Eve, offering him a free trip to Bahamas to steal his personal
```

information. 
<!--
#### Different types of phishing

1. Email Phishing scams
2. Website Phishing scams
3. Text Phishing scams

--->
## A closer look

Alice helps Bob understand the red flags in his email by taking a detailed look! 

> <i>Note: There are times when you need to provide personal information to a trusted reciepent.
> <br> However,
> providing personal information can also lead to phishing. In this workshop we will understand how to differentiate between a legitimate email and a scam email.  </i>

Reviewer: ```suggestion
_Note: There are times when you need to provide personal information to a trusted recipient.
<br> However,
providing personal information can also lead to phishing. In this workshop we will understand how to differentiate between a legitimate email and a scam email._
```
